integration-tests/src/liquidation.rs

```diff
@@ -25,7 +29,8 @@ use sp_core::{H256, U256};
 use sp_runtime::{traits::CheckedConversion, SaturatedConversion};
 
 // ./target/release/scraper save-storage --pallet EVM AssetRegistry Timestamp Omnipool Tokens MultiTransactionPayment EmaOracle Balances --uri wss://rpc.nice.hydration.cloud:443
-pub const PATH_TO_SNAPSHOT: &str = "evm-snapshot/SNAPSHOT";
+//pub const PATH_TO_SNAPSHOT: &str = "evm-snapshot/SNAPSHOT";
+pub const PATH_TO_SNAPSHOT: &str = "snapshots/hsm/SNAPSHOT";
```

Comment:
Why did you change the snapshot? Also you left there details about previous snapshot. Did you use the same command to get new snapshot? If there is no reason for a new snapshot, please revert in back. I use it in my PR.

Reply:
yes, i was just testing the snapshot...now there is no change to liquidation file in integration tests.